Tests/ChainsafeRPCTests.cs

```diff
@@ -12,52 +11,56 @@ namespace Tests
+            _nftAbi =
```

Comment:
the ABI is almost the same as before, it only adds the safeMint function because now we are using the Ganache instance for those tests